tracer/src/Datadog.Trace/Activity/Handlers/ActivityHandlerCommon.cs

```diff
@@ -163,6 +163,17 @@ public static void ActivityStarted<T>(string sourceName, T activity, OpenTelemet
                 }
 
                 activityKey ??= activity.Id;
+
+                if (activityKey is null)
+                {
+                    // identified by Error Tracking
+                    // unsure how exactly this occurs after reading through the Activity source code
+                    // Activity.Id, Activity.SpanId and/or Activity.TraceId were null
+                    // if this is the case, just ignore the Activity
+                    activityMapping = default;
```

Comment:
Thoughts on adding logging here of some sort?

Warning level?

Reply:
> Thoughts on adding logging here of some sort

_Maybe_. My concern is that we don't understand why this happens, so we'll be writing a lot of logs, which noone will see 😬